package.json

```diff
@@ -1,13 +1,14 @@
 {
-  "name": "@cromefire_/nativeshell-api-definition",
-  "version": "1.0.0",
+  "name": "nativeshell-stub",
+  "version": "0.1.0",
   "module": "dist/index.js",
   "types": "dist/index.d.ts",
   "license": "Apache-2.0",
   "scripts": {
     "build": "tsc -b tsconfig.json",
     "lint": "eslint src/**",
-    "docs": "typedoc"
+    "docs": "typedoc",
+    "type2docfx": "typedoc --json api.json && npx type2docfx api.json yml"
```

Comment:
No need for npx here

Reply:
I get the following without using it:

`/usr/bin/env: ‘node\r’: No such file or directory`

Reply:
Is the 2nd command a non node command?

Reply:
It is a Node command. Source is here: https://github.com/docascode/type2docfx

Reply:
Then please use https://www.npmjs.com/package/npm-run-all for cross platform, I think && makes problems with windows